threejs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zestymarket/threejs-sdk",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
Will this break other projects that use an older version of threejs? Might want to check with hyperfy too I believe they rely on threejs.

Reply:
Ash gave his okay, we're good to make this change